--- bluez-alsa-master/configure.ac.orig	2019-01-26 17:02:28.000000000 +0100
+++ bluez-alsa-master/configure.ac	2019-02-20 16:57:31.569967900 +0100
@@ -113,7 +113,7 @@
 AM_CONDITIONAL([ENABLE_HCITOP], [test "x$enable_hcitop" = "xyes"])
 AM_COND_IF([ENABLE_HCITOP], [
 	PKG_CHECK_MODULES([LIBBSD], [libbsd])
-	PKG_CHECK_MODULES([NCURSES], [ncurses])
+	PKG_CHECK_MODULES([NCURSES], [ncursesw])
 ])
 
 AC_ARG_ENABLE([test],
