diff -Naur ladspa-sdk-1.13.orig/Makefile ladspa-sdk-1.13/Makefile
--- ladspa-sdk-1.13.orig/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ ladspa-sdk-1.13/Makefile	2019-02-22 13:14:49.120133000 +0100
@@ -0,0 +1,5 @@
+all:
+	$(MAKE) -C src targets
+
+install:
+	$(MAKE) -C src install
diff -Naur ladspa-sdk-1.13.orig/src/makefile ladspa-sdk-1.13/src/makefile
--- ladspa-sdk-1.13.orig/src/makefile	2019-02-22 13:19:10.635496100 +0100
+++ ladspa-sdk-1.13/src/makefile	2019-02-22 13:14:57.441015900 +0100
@@ -4,9 +4,9 @@
 #
 # Change these if you want to install somewhere else.
 
-INSTALL_PLUGINS_DIR	=	/usr/lib/ladspa/
-INSTALL_INCLUDE_DIR	=	/usr/include/
-INSTALL_BINARY_DIR	=	/usr/bin/
+INSTALL_PLUGINS_DIR	?=	$(DESTDIR)/usr/lib/ladspa/
+INSTALL_INCLUDE_DIR	?=	$(DESTDIR)/usr/include/
+INSTALL_BINARY_DIR	?=	$(DESTDIR)/usr/bin/
 
 ###############################################################################
 #
@@ -28,6 +28,7 @@
 			../bin/listplugins
 CC		?=	cc
 CXX		?=	c++
+LDFLAGS		?=	$(LDFLAGS) -shared
 
 WITH_INIT 	= noise amp delay filter
 
@@ -98,19 +99,19 @@
 	$(CC) $(CFLAGS)							\
 		-o ../bin/applyplugin					\
 		applyplugin.o load.o default.o				\
-		$(LIBRARIES) $(LDFLAGS)
+		-shared $(LIBRARIES) $(LDFLAGS)
 
 ../bin/analyseplugin:	analyseplugin.o load.o default.o
 	$(CC) $(CFLAGS) 						\
 		-o ../bin/analyseplugin 				\
 		analyseplugin.o load.o default.o			\
-		$(LIBRARIES) $(LDFLAGS)
+		-shared $(LIBRARIES) $(LDFLAGS)
 
 ../bin/listplugins:	listplugins.o search.o
 	$(CC) $(CFLAGS) 						\
 		-o ../bin/listplugins	 				\
 		listplugins.o search.o					\
-		$(LIBRARIES) $(LDFLAGS)
+		-shared $(LIBRARIES) $(LDFLAGS)
 
 ###############################################################################
 #
