--- upmpdcli-1.4.1/Makefile.am.orig	2019-02-28 10:11:01.908300300 +0100
+++ upmpdcli-1.4.1/Makefile.am	2019-02-28 10:11:28.779826100 +0100
@@ -241,9 +241,9 @@
 install-data-hook: web.tar.gz rdpl2stream.tar.gz uprclbottle.tar.gz
 	(cd $(DESTDIR)$(sysconfdir); test -f upmpdcli.conf || \
 		mv upmpdcli.conf-dist upmpdcli.conf; chmod 600 upmpdcli.conf)
-	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < web.tar.gz 
-	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < rdpl2stream.tar.gz
-	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < uprclbottle.tar.gz
+	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < $(srcdir)/web.tar.gz 
+	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < $(srcdir)/rdpl2stream.tar.gz
+	(cd $(DESTDIR)$(pkgdatadir); tar xzf -) < $(srcdir)/uprclbottle.tar.gz
 	(cd $(DESTDIR)$(pkgdatadir); test -d src_scripts || mkdir src_scripts)
 	(cd $(DESTDIR)/$(cdpluginstidaldir); chmod a+x tidal-app.py) 
 	(cd $(DESTDIR)/$(cdpluginsqobuzdir); chmod a+x qobuz-app.py) 
